packages/components/src/templates/dialog.stories.ts

```diff
@@ -0,0 +1,150 @@
+              respective fiscal year, adjusted for the distribution included in the previous fiscal year.
+            </p>
+            <p class="sd-paragraph">
+              The Sustainable Finance Disclosure Regulation (SFDR) regulates disclosure requirements for financial
+              products. Art. 6 products do not take sustainability into account, Art. 8 promotes ecological and social
+              characteristics. Art. 9 products have sustainable investments as their investment objective.
+            </p>
+            <p class="sd-paragraph">
+              Any front-end load is fully passed on to the fund’s intermediary. If you subscribe directly with Union
+              Investment, this does not apply.
+            </p>
+            <p class="sd-paragraph">
+              Please read the prospectus and the key information document before making a final investment decision.
+              These contain detailed product-specific information, particularly on investment objectives, investment
+              principles, opportunities and risks, as well as explanations of the fund’s risk profile. These documents,
+              as well as the investment conditions and the annual and semi-annual reports, form the sole binding basis
+              for the purchase of the fund. They are available free of charge in German from Union Investment
+              Institutional GmbH, Weißfrauenstraße 7, 60311 Frankfurt am Main, Tel. +49 69 2567-7652, Fax +49 69
+              2567-1010,
+              <sd-link href="http://institutional@union-investment.de">institutional@union-investment.de</sd-link>,
+              <sd-link href="http://www.institutional.union-investment.de"
+                >www.institutional.union-investment.de</sd-link
+              >. If the document concerns a sustainable fund, you will find information on the sustainability-relevant
+              aspects of the fund at
+              <sd-link href="https://ui-link.de/fondspreisseite">https://ui-link.de/fondspreisseite</sd-link>.
+            </p>
+            <p class="sd-paragraph">
+              A summary of your investor rights in German and further information on collective redress mechanisms can
+              be found under “Notices and Complaints” at
+              <sd-link href="https://ui-link.de/richtlinien">https://ui-link.de/richtlinien</sd-link>. The respective
+              fund-issuing company may decide at any time to revoke any arrangements it may have made for the
+              distribution of shares of a fund and/or share classes of a fund in a member state other than its home
+              member state, subject to the conditions of Art. 93a of Directive 2009/65/EC and Art. 32a of Directive
+              2011/61/EU.
+            </p>
+          </div>
+        </sd-scrollable>
+        <div slot="footer" class="flex justify-end w-full space-x-2">
+          <sd-button variant="primary" size="lg" type="button"> Accept terms of use </sd-button>
```

Comment:
Pls change order - The primary button in a group always shows up on the right-hand side:
![Bildschirmfoto 2024-08-28 um 10 52 41](https://github.com/user-attachments/assets/9d910954-86d1-441c-aadc-4dd357cff74e)

Reply:
Updated